server.go

```diff
@@ -219,17 +230,22 @@ func (server *Server) listenProviders(stop chan bool) {
 			return
 		case configMsg, ok := <-server.configurationChan:
 			if !ok {
+				server.backendReloadCounter.With("state", "failed", "type", configMsg.ProviderName).Add(1)
 				return
 			}
+			var labels []string
 			server.defaultConfigurationValues(configMsg.Configuration)
 			currentConfigurations := server.currentConfigurations.Get().(configs)
 			jsonConf, _ := json.Marshal(configMsg.Configuration)
 			log.Debugf("Configuration received from provider %s: %s", configMsg.ProviderName, string(jsonConf))
 			if configMsg.Configuration == nil || configMsg.Configuration.Backends == nil && configMsg.Configuration.Frontends == nil {
 				log.Infof("Skipping empty Configuration for provider %s", configMsg.ProviderName)
+				labels = []string{"state", "failed", "type", configMsg.ProviderName}
```

Comment:
The only parameter that changes between the different occasions where we assign the slice is the state. Can we initialize the slice with the other parameters in line 236 right away and just complete the state in the subsequent lines, respectively?